src/utils/hash.ts

```diff
@@ -1,62 +1,77 @@
 import { get, post } from "./http";
+import { storeHashedValues, getStoredHashedValues } from "../utils/file";
 
 const _sodium = require('libsodium-wrappers');
 let sodium: any;
 
-let userHashedValues: any = {};
-
-export async function hashValue(value: string, dataType: string, jwt?: string) {
-  if (!jwt) {
-    return ""
-  }
-
-  if (!value) {
-    return value;
-  }
+let lastJwt: string = "";
 
+export async function hashValues(payload: any, jwt: string) {
   if (sodium === undefined) {
-    await Promise.all([setUserHashedValues(jwt), _sodium.ready]).then(() => {
+    await Promise.all([_sodium.ready]).then(() => {
```

Comment:
do we still need the Promise.all for _sodium.ready? it works but just thinking it would be fine to just await on it since there's only a single await now